app/src/main.js

```diff
@@ -27,9 +26,22 @@ global.View = function() {
     view.appendChild(spinner.el);
   }
 
-  var installationFaild = function(error) {
+  var installationFailed = function(error) {
+    var errors = {
+      175: 'Installation failed because you do not have administrator access.',
+      201: 'Installation failed because the extension invalid.',
+      411: 'Installation failed because the extension is not compatible with the installed applications.',
+      407: 'Installation failed because this extension requires another extension.',
+      408: 'Installation failed because this extension requires another extension.',
+      412: 'Installation failed because an extension of the same name exists.',
+      418: 'Installation failed because a newer version of the extension is installed.',
+      456: 'Installation failed because Adobe applications are running',
+      458: 'Installation failed because none of the required applications are installed',
+      459: 'Installation failed because the extension is not compatible with the installed applications.'
+    };
+
     view.removeChild(spinner.el);
-    $(view).find('.status').html(error);
+    $(view).find('.status').html(errors[error] || 'Error: ' + error);
```

Comment:
This will print one of the errors above if an error code matches, and will log anything else as `Error: <code>` if there isn't a match.